WebGoatCore/Controllers/CheckoutController.cs

```diff
@@ -18,7 +18,7 @@ public class CheckoutController : Controller
         private readonly CustomerRepository _customerRepository;
         private readonly ShipperRepository _shipperRepository;
         private readonly OrderRepository _orderRepository;
-        private CheckoutViewModel _model;
+        private CheckoutViewModel _model = null!;
```

Comment:
I am not very happy because of that, because we will create an object which will never be used just to supress compilation warning. But ok... I will create it in ctor.

Reply:
I think creating it in constructor would be ok, but I'm really concerned about the HttpContext.Session (and potentially others) - that could change program semantics

Reply:
I don't know what was the original issue with not initializing the field. Guessing: if it was something about non-nullable field not being initialized in constructor, then maybe a better solution would be to make it nullable?
`private CheckoutViewModel? _model;`

Reply:
Making it nullable will cause even more warnings. I think the checkout is done late in application lifecycle, so I do not see danger regarding HttpSession. Fixed by creating instance in constructor.

Reply:
See my comment to the new change, it won't work because it won't call InitializeModel()